qemu/tests/qemu_guest_agent.py

```diff
@@ -184,14 +207,22 @@ def _get_latest_pkg(self):
                                               tag_version)
         get_latest_mdl_tag_cmd = "brew list-targets |grep"
         get_latest_mdl_tag_cmd += " %s |sort -r |head -n 1" % platform_tag
-        latest_mdl_tag = process.system_output(get_latest_mdl_tag_cmd,
-                                               shell=True,
-                                               timeout=query_timeout
-                                               ).strip().split()[0].decode()
-        error_context.context("Get qemu-guest-agent rpm pkg url.",
-                              logging.info)
+        mdl_tag, build_s = get_mdl_tag_build_status(get_latest_mdl_tag_cmd)
+        logging.info("mdl_tag is %s, build_s is %s" % (mdl_tag, build_s))
+
+        if not build_s:
```

Comment:
How about add one more info, if the build_s is false, to explain it is tring to get the pre build. Since show L222 after L211 seems strange.

Reply:
ok, modify, could you check if it's better now?